main.go

```diff
@@ -137,7 +137,9 @@ func newNewCmd(arguments []string) *newCmd {
 	flags := flag.NewFlagSet("pushup new", flag.ExitOnError)
 	moduleNameFlag := newRegexString(`^\w[\w-]*$`, "example/myproject")
 	flags.Var(moduleNameFlag, "module", "name of Go module of the new Pushup app")
-	flags.Parse(arguments)
+	if err := flags.Parse(arguments); err != nil {
```

Comment:
I think we should leave the regular `flags.Parse(arguments)` here - the error return for `flag.FlagSet.Parse()` doesn't say anything about whether the args could be parsed or not:

> The return value will be ErrHelp if -help or -h were set but not defined.

https://cs.opensource.google/go/go/+/refs/tags/go1.19.5:src/flag/flag.go;l=1101

So I think it's fine to treat it like the API signature of `flag.Parse()` which has no return type.